### DISCUSSION TITLE
NextJs SDK: currentUser() does not return full User object (Organization fields)

### DISCUSSION THREAD

**Discussion by brianwatroba** [clerk/javascript#2140] [Support request - DO NOT OPEN HERE]
**Background:** Using NextJs 14, I'm trying to fetch the current user with `currentUser()` from a Server Component. [Link to currentUser() docs:](https://clerk.com/docs/references/nextjs/current-user).

**Issue:** The docs say that `currentUser()` should return the `User` object, but it's missing all fields about Organizations.

I tried using the backend sdk to fetch info about the user, but also nothing re: Organzations.

The only way I've managed to retrieve Organization data for a userId is the client-side hook `useUser()`, which has `organizationMemberships` field. How can I access this from back end?

Am I missing something? Thanks!

**Comment by brianwatroba** [comment]
Asking in the Discord instead

**Reply by sunlee-newyork** [reply]
Did you get an answer?

**Reply by ryanyue123** [reply]
also seeing this now
